addons/mail/static/src/components/chat_window_manager/chat_window_manager_tests.js

```diff
@@ -364,6 +364,10 @@ QUnit.test('open chat from "new message" chat window should open chat in place o
         "Partner 131",
         "autocomplete suggestion should target the partner matching search term"
     );
+    assert.ok(
+        link.classList.contains('ui-state-active'),
+        "first autocomplete result should be automatically selected"
+    );
```

Comment:
you could use `assert.hasClass` for this, it will be easier to read and give better error message in case of error